go/runtime/history/indexer.go

```diff
@@ -15,6 +15,37 @@ import (
 	roothash "github.com/oasisprotocol/oasis-core/go/roothash/api"
 )
 
+// IndexerStatus is the concise status of runtime history indexer.
+type IndexerStatus string
+
+const (
```

Comment:
```suggestion
const (
	StatusStarted     = "started"
	StatusReindexing   = "reindexing"
	StatusIndexing     = "indexing"
	StatusStopped     = "stopped"
)
```
So that you know what is the right order.